### ISSUE TITLE
Add missing `aria-label` attribute to the `BitLabel` component

### ISSUE THREAD

**Issue by Cyrus-Sushiant** [bitfoundation/bitplatform#7880]
To improve accessibility, we need to add the `aria-label` attribute to the `BitLabel` component. This will ensure that screen readers can provide a meaningful description of the component for users with disabilities.
